javascript-objects/main.js

```diff
@@ -0,0 +1,52 @@
+// Student Object
+
+var StudentObj = {
```

Comment:
you want your variables to be lower case good sir. Uppercase variable names are usually reserved for class definitions which we will be going over at a later date!